Gemfile

```diff
@@ -6,12 +6,13 @@ gem 'protected_attributes'
 
 gem 'rails', '~> 4.1.4'
 gem 'pg', '~> 0.18'
-gem 'unicorn'
+#gem 'unicorn'
```

Comment:
delete this please. As a rule, don't leave commented out code. We have git for that.